addons/mail/wizard/mail_wizard_invite.py

```diff
@@ -58,30 +60,19 @@ def add_followers(self):
 
             model_name = self.env['ir.model']._get(wizard.res_model).display_name
             # send an email if option checked and if a message exists (do not send void emails)
-            if wizard.send_mail and wizard.message and not wizard.message == '<br>':  # when deleting the message, cleditor keeps a <br>
-                message = self.env['mail.message'].create({
-                    'subject': _('Invitation to follow %(document_model)s: %(document_name)s', document_model=model_name, document_name=document.display_name),
-                    'body': wizard.message,
-                    'record_name': document.display_name,
-                    'email_from': email_from,
-                    'reply_to': email_from,
-                    'model': wizard.res_model,
-                    'res_id': wizard.res_id,
-                    'reply_to_force_new': True,
-                    'email_add_signature': True,
-                })
-                email_partners_data = []
-                recipients_data = self.env['mail.followers']._get_recipient_data(document, 'comment', False, pids=new_partners.ids)[document.id]
-                for _pid, pdata in recipients_data.items():
-                    pdata['notif'] = 'email'
-                    email_partners_data.append(pdata)
-
-                document._notify_thread_by_email(
-                    message, email_partners_data,
-                    send_after_commit=False
+            if wizard.send_notification and wizard.message and not wizard.message == '<br>':  # when deleting the message, cleditor keeps a <br>
+                document.message_notify(
+                    subject=_('Invitation to follow %(document_model)s: %(document_name)s',
+                              document_model=model_name, document_name=document.display_name),
+                    body=wizard.message,
+                    record_name=document.display_name,
+                    email_from=email_from,
+                    reply_to=email_from,
+                    model=wizard.res_model,
+                    res_id=wizard.res_id,
+                    reply_to_force_new=True,
+                    email_add_signature=True,
+                    partner_ids=new_partners.mapped('id'),
+                    message_type='comment',
```

Comment:
Notifications should always have the "user_notification" message_type as they have some specific check in access rules and display. Is there any reason to try to force comment ? To be complete, I have a commit somewhere that prevent from calling message_notify with a custom message_type :D .

Reply:
No it is an error, it is indeed a user notification. As it is the default value, I have removed the parameter.

Reply:
No it is an error, it is indeed a user notification. As it is the default value, I have removed the parameter.